bigquery_etl/shredder/config.py

```diff
@@ -151,6 +151,7 @@ def fields(self) -> Tuple[str, ...]:
 context_id_target = partial(DeleteTarget, field=CONTEXT_ID)
 
 DELETE_TARGETS = {
+    client_id_target(table="search_derived.acer_cohort_v1"): DESKTOP_SRC,
```

Comment:
I assumed since this table includes client_ids we also should add it here -- is that right? At any rate, anyone whose data is deleted will no longer be in clients_daily so there's no value in keeping their client_ids around

Reply:
Good catch, I haven't thought about this. After looking at the Shredder code I agree but just to be sure would love a confirmation from @relud.

Reply:
yes